gnucash/import-export/bi-import/dialog-bi-import.c

```diff
@@ -526,9 +526,9 @@ gnc_bi_import_create_bis (GtkListStore * store, QofBook * book,
     GtkWidget *dialog;
     time64 today;
     InvoiceWindow *iw;
-    gchar *new_id = NULL;
     gint64 denom = 0;
     gnc_commodity *currency;
+    GString *running_id;
```

Comment:
Well, that is just my lack of understanding. I used `GString*` because the original author used the same for a similar purpose in the current version of this file, line 234:

`GString *prev_id, *prev_date_opened, *prev_date_posted, *prev_owner_id, *prev_date;	// needed to fix multi line invoices`

Reply:
That's fair enough.